app/api/canvas/canvas.api.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
```

Comment:
The naming you have of files is a well adopted conversation, so no issue, but our convention is that we don't postfix file-names with the type, like canvas.api.js, since it's already under the api-folder. So canvas.js should be enough.